bin/elmstatic.js

```diff
@@ -399,25 +401,48 @@ function generateEverything(pages, posts, options) {
     const dotGitContent = Fs.pathExistsSync(dotGitPath) ?
         Fs.readFileSync(Path.join(config.outputDir, ".git")).toString() : null
 
-    log(`  Cleaning out the output path (${config.outputDir})`)
-    Fs.emptyDirSync(config.outputDir)
+    if (options.keepOutputAlive) {
+        ; // do nothing
+    } else {
+        log(`  Cleaning out the output path (${config.outputDir})`)
+        Fs.emptyDirSync(config.outputDir)
+    }
+
 
     if (R.is(String, dotGitContent))
         Fs.writeFileSync(dotGitPath, dotGitContent)
     else
         ; // Do nothing, no .git file existed
 
     log("  Writing HTML")
-    R.forEach(writeHtmlPage, newPages)
-    R.forEach(writeHtmlPage, newPosts)
-    R.forEach(writeHtmlPage, tagPages)
+    
+    Promise.all(
+        [].concat(R.map(writeHtmlPage, newPages), R.map(writeHtmlPage, newPosts), R.map(writeHtmlPage, tagPages))
+    )
+    .then(promiseFileWritings => {
+        log("  Duplicating pages")
+        duplicatePages(config.copy, config.outputDir)
+        log("  Generating feeds")
+        generateFeeds(config.feed, config.outputDir, R.reject(R.propEq("isIndex", true), newPosts))
+
+        if (config.postProcess.length > 0) {
```

Comment:
I think the best way to check this would be `!R.isNil(config.postProcess) && !R.isEmpty(config.postProcess)`.

Reply:
I will change it accordingly, but I also think this is not needed anymore, if the output dir is not deleted anymore ...